packages/core/src/components/Tipseen/__stories__/tipseen.stories.helpers.js

```diff
@@ -15,6 +15,14 @@ export const modifiers = [
   }
 ];
 
+export const wizardContent = [
+  <div>Message for the user will appear here, to give more information about the feature.</div>,
```

Comment:
Do we maybe want to change the text a bit between steps to make it seem like it's changing and not remaining static?

Reply:
Theoretically yes, but in this specific example nothing happens if the buttons are pressed so it doesn't really matter